src/components/projects/CreateProjectForm.vue

```diff
@@ -572,30 +568,10 @@ export default {
                 "Red",
                 "Green",
                 "Blue",
-                "NIR",
-                "O3",
                 "HA",
-                "N2",
+                "O3",
                 "S2",
-                "ME",
-                "air",
-                "w",
-                "clear",
-                "silica",
-                "up",
-                "gp",
-                "rp",
-                "ip",
-                "zs",
-                "zp",
-                "Y",
-                "U",
-                "B",
-                "V",
-                "Rc",
-                "Ic",
             ],
-                
```

Comment:
Just to clarify - is the generic filter list a list of filters that all sites have, or is it a list of filters that many sites have? If the latter, is it possible for a user to schedule a project using one of these filters at a site that doesn't have that filter, or is there something that prevents that?

Reply:
The generic filter list is supposed to be a short list of basic filters that all sites have ('red', 'green', etc.). However, there are still some filter name inconsistencies across site config files, so we'll have to decide on a dictionary of acceptable names or something similar and stick with it. Right now, nothing in the project form prevents a user from scheduling a project with a generic filter if the site doesn't have it in the config file.